packages/web-app-files/src/components/SideBar/Shares/SharesPanel.vue

```diff
@@ -39,23 +38,15 @@ export default defineComponent({
     const sharesStore = useSharesStore()
     const { loading: sharesLoading } = storeToRefs(sharesStore)
 
-    const { incomingParentShare, ...rest } = useIncomingParentShare()
-    provide('incomingParentShare', incomingParentShare)
-
     return {
-      incomingParentShare,
-      ...rest,
       sharesLoading,
-      resource: inject<Resource>('resource'),
+      resource: inject<Ref<Resource>>('resource'),
       activePanel: inject<String>('activePanel')
     }
   },
   watch: {
     sharesLoading: {
       handler: function (sharesLoading, old) {
-        if (!sharesLoading) {
-          this.loadIncomingParentShare.perform(this.resource)
-        }
         // FIXME: !old can be removed as soon as https://github.com/owncloud/web/issues/7621 has been fixed
```

Comment:
heads up: this was closed meanwhile, could be resolved (in a follow up)

Reply:
Added to the checks in https://github.com/owncloud/web/issues/10418.